src/AppLayout.tsx

```diff
@@ -27,29 +30,118 @@ interface IAppLayout {
 export const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
   const AppLogoImage = () => <img src={pncLogoText} alt="Newcastle Build System" />;
 
-  const AppHeaderTools = (
-    <PageHeaderTools>
-      <PageHeaderToolsGroup>
-        <PageHeaderToolsItem>
-          <Button variant={ButtonVariant.plain}>
-            <CogIcon />
-          </Button>
-        </PageHeaderToolsItem>
-        <PageHeaderToolsItem>
-          <Button variant={ButtonVariant.plain}>
-            <OutlinedQuestionCircleIcon />
-          </Button>
-        </PageHeaderToolsItem>
-        <PageHeaderToolsItem>
-          <Button variant={ButtonVariant.plain}>
-            <BellIcon />
-          </Button>
-        </PageHeaderToolsItem>
-      </PageHeaderToolsGroup>
-    </PageHeaderTools>
-  );
+  const headerConfigDropdownItems = [
+    <DropdownItem key="variables" href="/admin/variables">
+      Variables
+    </DropdownItem>,
+    <DropdownItem key="administration" href="/admin/administration">
+      Administration
+    </DropdownItem>,
+  ];
+
+  const headerQuestionDropdownItems = [
+    <DropdownItem key="about" href="/pnc-web/about">
+      About
+    </DropdownItem>,
+    <DropdownItem key="users guide" href="https://docs.engineering.redhat.com/display/JP/User%27s+guide">
+      User's guide
+    </DropdownItem>,
+  ];
+
+  const headerUserDropdownItems = [<DropdownItem key="logout">Logout</DropdownItem>];
+
+  const AppHeaderTools = () => {
+    const [isHeaderConfigOpen, setIsHeaderConfigOpen] = useState(false);
+    const [isHeaderQuestionOpen, setIsHeaderQuestionOpen] = useState(false);
+    const [isHeaderUserOpen, setIsHeaderUserOpen] = useState(false);
+    const [currentUser, setCurrentUser] = useState<any>(null);
+
+    return (
+      <PageHeaderTools>
+        <PageHeaderToolsGroup>
+          <PageHeaderToolsItem isSelected={true}>
+            <Dropdown
+              onSelect={() => {
```

Comment:
isn't using `onToggle` method enough? why the same code needs to be executed also here?